util/slack.py

```diff
@@ -118,53 +118,13 @@ def get_linger_time_message_and_attachment(self, story, stats_per_slug, time_buc
                 time
             )
 
-        if time_bucket == 'hour 8':
-            message = ("%s hours in and _%s_ had users spending about *%s* "
-                "interacting with the graphic.") % (
+        if time_bucket == '8 hours' or time_bucket == '12 hours' or time_bucket == 'A day' or time_bucket == 'A day and a half' or time_bucket == 'Two days' or time_bucket == 'Two days and a half':
```

Comment:
If this covers all scenarios after the first message, you could reduce this to an "else" statement. Eg:

```
if time_bucket == 'hour 4':
    # Send "I just started tracking" message. 

else: 
    # The new generic message
```